run.py

```diff
@@ -60,6 +61,10 @@
                                      app.config['DEBUG'])
 log = log_configuration.create_logger("Platform")
 
+# Create bucket objext using GCS storage client
+sa_file = os.path.join(app.config.get('INSTALL_FOLDER', ''), app.config.get('SERVICE_ACCOUNT_FILE', ''))
+storage_client = storage.Client.from_service_account_json(sa_file)
+storage_client_bucket = storage_client.bucket(app.config.get('GCS_BUCKET_NAME', ''))
```

Comment:
Similar to the above, if the configs are missing for any of these, we should throw a fatal error and not start the application.